paywall/src/unlock.js/setupUnlockProtocolVariable.ts

```diff
@@ -23,16 +32,28 @@ export default function setupUnlockProtocolVariable(
   const hideCheckoutModal = () => {
     hideIframe(window, CheckoutUIIframe)
   }
+  const getState = () => lockStatus
```

Comment:
Could you add a note that lockStatus needs to be private and can only updated by the event listener?